decode_largedata_test.go

```diff
@@ -0,0 +1,93 @@
+package hessian
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+)
+
+func TestDecodeLargeData(t *testing.T) {
```

Comment:
change the test to Benchmark  instead

Reply:
I think this pr is a general performance improvement , it only needs a simple struct decode benchmark to prove that.